react/src/js/components/Consonant/Infobit/Type/Button.jsx

```diff
@@ -22,6 +22,8 @@ const buttonType = {
     isCta: bool,
     onFocus: func,
     title: string,
+    tabIndex: string,
```

Comment:
PropTypes declares `tabIndex: string` with a default of `''`, but DOM `tabIndex` is almost always numeric (0 or –1). 

Not sure we should have an empty-string fallback for this (since I think this will create invalid HTML)

Ideally PropType would be `number` and default to `0`.